<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="directory_label">
          <property name="text">
           <string>Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="directory_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="open_button">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fps_label">
          <property name="text">
           <string>FPS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fps_spinbox">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="convert_button">
          <property name="text">
           <string>Convert to video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="compression_label">
          <property name="text">
           <string>Image compression</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="compression_spinbox">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
